pruning/README.md

```diff
@@ -0,0 +1,29 @@
+# Pruning
+
+## Overview
+
+Pruning is the mechanism for deleting old heights from the disk. Depending on the use case,
+nodes may require different pruning strategies. For example, archive nodes must keep all
+the states and prune nothing. On the other hand, a regular validator node may want to only keep 100 latest heights for performance reasons.
+
+## Strategies
+
+The strategies are configured in `app.toml`:
+pruning = "< strategy >" # where the options are:
+- `default`: only the last 100,000 states(approximately 1 week worth of state) are kept; pruning at 100 block intervals
```

Comment:
We changed the default value some time ago. Did you change it back to 100k in this PR?

Reply:
We have 100k in Osmosis. I missed this difference when cherry-picking. Changed back to the updated default value of 362880.

What was the reason for this change?